tests/cli/main.cpp

```diff
@@ -446,6 +446,112 @@ BOOST_FIXTURE_TEST_CASE( cli_vote_for_2_witnesses, cli_fixture )
    }
 }
 
+///////////////////////
+// SON CLI
+///////////////////////
+BOOST_FIXTURE_TEST_CASE( create_son, cli_fixture )
```

Comment:
Add a unique message at the start and end of the test so that QA and community can easily verify the results by looking at the logs.